SpatialGDK/Source/SpatialGDK/Private/Utils/ActorGroupManager.cpp

```diff
@@ -0,0 +1,64 @@
+#include "Utils/ActorGroupManager.h"
+#include "SpatialGDKSettings.h"
+
+void UActorGroupManager::Init()
+{
+	if (const USpatialGDKSettings* Settings = GetDefault<USpatialGDKSettings>())
+	{
+		DefaultWorkerType = Settings->DefaultWorkerType.WorkerTypeName;
+		if (Settings->bEnableOffloading)
+		{
+			for (const TPair<FName, FActorGroupInfo> ActorGroup : Settings->ActorGroups)
```

Comment:
```suggestion
			for (const auto& ActorGroup : Settings->ActorGroups)
```

Reply:
???

Reply:
My Bad. I think with TMap iteration, it's nice to be able to see what types the key and value is.